treetime/distribution.py

```diff
@@ -94,7 +94,7 @@ def multiply(dists):
             delta_dist = dists[delta_dist_ii]
             new_xpos = delta_dist.peak_pos
             new_weight  = np.prod([k.prob(new_xpos) for k in dists if k!=delta_dist_ii]) * delta_dist.weight
-            res = Distribution.delta_function(new_xpos, weight = new_weight,min_width=min_width)
+            res = Distribution.delta_func_logtion(new_xpos, weight = new_weight,min_width=min_width)
```

Comment:
Is the `logtion` in `delta_func_logtion` on purpose or a find/replace mistake and should be `delta_func_log`? :)